packages/rest/src/rest.server.ts

```diff
@@ -629,6 +630,20 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
   }
 
+  /**
+   * Redirect route by invoking a handler function.
+   *
+   * ```ts
+   * server.redirect('/explorer', '/explorer/');
+   * ```
+   *
+   * @param source URL path of the redirect endpoint
+   * @param target URL path of the endpoint
+   */
+  redirect(source: string, target: string): Binding {
+    return this.route(new RedirectRoute(source, this._basePath + target));
```

Comment:
> I find this surprising. Why are we prepending `this._basePath` to `target`, but not to `source`? I would expect basePath to be prepended to both or to none. What am I missing?
>
1. adding the the base path to the source will cause to double the base path so the result will be 
something like basePath/basePath/redirectRoute.
2. the reason of adding base path to source to avoid 404 not found cause the location header not holding the base path so when the request come back during the redirect process express/lb4 router will return 404.   

eventually  i wanna to achieve to redirect 
route if not exist this will help developers and give them some flexibility of using redirect api

lets assume i have configured the lb4 app like this ; 
```ts
    restApp.basePath('/api');
    await restApp.controller(barController).lock();
    await restApp.controller(fooController).lock();
    restApp.restServer.redirect('/foo','/bar');
    restApp.restServer.redirect('/notExistRoute','/bar');
    await restApp.start();
```

in the above example the user will not figure out what happening behind the scene , removing the base path to be added as target `/api/bar` a litile bit confusing me . 

the result of requesting `/api/foo` and `/api/notExistRoute` it will be the bar controller response. 
 even when the `api/notExistRoute` missing as route .